mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -451,6 +439,8 @@ class ALSModel private[ml] (
 @Since("1.6.0")
 object ALSModel extends MLReadable[ALSModel] {
 
+  @transient private[recommendation] val _f2jBLAS = new F2jBLAS
```

Comment:
Does this require significant initialization?  You could use org.apache.spark.ml.linalg.BLAS.f2jBLAS